typescriptSrc/interpreter.ts

```diff
@@ -143,11 +144,25 @@ module interpreter {
     }
 
     function callWorldStepper( vms : VMS ) : void {
-      const value = vms.getPendingNode().label().getVal();
+      const node = vms.getPendingNode();
+      const value = node.label().getVal();
       if (vms.getStack().hasField(value)) {
-        //let stepper = vms.getStepper(value);        
-        //stepper(vms);  
-      }
+        let field = vms.getStack().getField(value).getValue();
+        if (field instanceof BuiltInV) {
+          let stepper = field.getStepper();
+          const args : Array<Value> = [];
+          for (let i = 0; i < node.count(); i++) {
+            args.push(vms.getChildVal(i));
+          }          
+          stepper(vms, args);
+        } 
+        else {
+          vms.reportError("Field was not a BuiltInV");
```

Comment:
This is an error message that may be seen by the user.  I'd suggest something like: "Attempt to call a value that is neither a closure nor a built-in function".